backend/hitas/views/condition_of_sale.py

```diff
@@ -0,0 +1,153 @@
+import datetime
+from typing import Any, Optional
+
+from django.db.models import Prefetch, Q
+from enumfields.drf import EnumField, EnumSupportSerializerMixin
+from rest_framework import serializers
+
+from hitas.exceptions import HitasModelNotFound
+from hitas.models import Apartment, ApartmentSale, ConditionOfSale, Owner, Ownership
+from hitas.models.condition_of_sale import GracePeriod, condition_of_sale_queryset
+from hitas.utils import get_many_or_cached_prefetch, lookup_id_to_uuid
+from hitas.views.utils import HitasModelSerializer, HitasModelViewSet
+
+
+class MinimalApartmentSerializer(HitasModelSerializer):
+    class Meta:
+        model = Apartment
+        fields = [
+            "id",
+            "street_address",
+            "apartment_number",
+            "floor",
+            "stair",
+        ]
```

Comment:
Both `hitas.views.apartment_list.ApartmentListSerializer` and `hitas.views.apartment.ApartmentHitasAddressSerializer` have fields we don't need in the response for a condition of sale, or are missing some of the fields. We could create an inheritance hierarchy to not duplicate the setup, but since there isn't any business logic in the serializer, I think we can just leave it as is.